streampark-common/src/main/scala/org/apache/streampark/common/util/HadoopUtils.scala

```diff
@@ -160,7 +160,7 @@ object HadoopUtils extends Logger {
    * Automatically goes to $HADOOP_HOME/etc/hadoop to load the configuration.<br> We recommend the
    * second method, without copying the configuration files.<br> </pre>
    */
-  def hadoopConf: Configuration = Option(reusableConf).getOrElse {
+  def loadHadoopConf: Configuration = Option(reusableConf).getOrElse {
```

Comment:
This is a variable name and should not reveal any more info, Not recommended to modify

Reply:
> This is a variable name and should not reveal any more info, Not recommended to modify

“ should not reveal any more info”，does too much information have any impact in practice? I don't quite understand this。Would you mind explaining this to me more?